docs/Users_Guide/glossary.rst

```diff
@@ -11912,6 +11957,25 @@ METplus Configuration Glossary
 
      | *Used by:* WaveletStat
 
+   POINT2GRID_TIME_OFFSET_WARNING
```

Comment:
@georgemccabe the `time_offset_warning` config option only applies to MET tools that do a grid-to-grid comparison. Doing some grepping, I only see calls to  this [time_offset_warning(...)](https://github.com/dtcenter/MET/blob/7d44b8db48dc79a47059252e50291c21000e1de0/src/basic/vx_config/config_file.cc#L329) function from wavelet_stat, mode, series_analysis, pcp_combine, grid_stat, and wwmca.

So it definitely doesn't impact point2grid. Not sure how to handle this.